PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,10 +5,11 @@
 - make the pull request against the ["v0.6" branch](https://github.com/Bitmessage/PyBitmessage/tree/v0.6)
 - it should be possible to do a fast-forward merge of the pull requests
```

Comment:
This actually isn't necessary anymore, I found a way around it, this line can be removed.